database/README.md

```diff
@@ -29,7 +29,7 @@ Install [DBeaver Community](https://dbeaver.io/).
 These queries can be executed from the database.
 Connect to the database.
 ```bash
-$ kubectl exec -it [postgres_pod] -n [namespace] -- psql -h localhost -U [db_user] --password -p 5432 [db_name]
+$ kubectl exec -it [postgres_pod] -n kubearchive -- psql -h localhost -U [db_user] --password -p 5432 [db_name]
```

Comment:
To make it easier for the reader, we can provide a way to automatically get the db pod based on the current naming:
```suggestion
$ kubectl exec -it  $(kubectl get -n kubearchive pods --no-headers -o custom-columns=":metadata.name" | grep database) -n kubearchive -- psql -h localhost -U ps_user --password -p 5432 postgresdb
```

Reply:
Changed